src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -14,24 +14,26 @@ public enum CallType
         Method = 1,
         Set = 8,
     }
-    public sealed partial class Collection : System.Collections.ICollection, System.Collections.IList
+    public sealed partial class Collection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
```

Comment:
>System.Collections.IEnumerable [](start = 77, length = 30)

Why was this added?

Reply:
Looks like the public documentation states that this was supposed to be implementing it, I'm assuming it was just missing from the original ref assembly?

---
In reply to: [259048143](https://github.com/dotnet/corefx/pull/35474#discussion_r259048143) [](ancestors = 259048143)